packages/discord.js/src/managers/GuildMemberManager.js

```diff
@@ -266,7 +266,10 @@ class GuildMemberManager extends CachedManager {
     _data.roles &&= _data.roles.map(role => (role instanceof Role ? role.id : role));
 
     _data.communication_disabled_until =
-      _data.communicationDisabledUntil && new Date(_data.communicationDisabledUntil).toISOString();
+      // eslint-disable-next-line eqeqeq
+      _data.communicationDisabledUntil != null
+        ? new Date(_data.communicationDisabledUntil).toISOString()
+        : _data.communicationDisabledUntil;
```

Comment:
Why is this change needed? 0 is not a value that is accepted here?

Reply:
Please read the description of the PR

Reply:
would `_data.communicationDisabledUntil ??= ...` work here ?

Reply:
That would only work if the value was nullish, and &&= wouldn't either because of 0

Reply:
oh yes, im reading the logic in reverse MB